hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson

```diff
@@ -81,6 +81,7 @@
     {
       name: sysrst_ctrl_ultra_low_pwr
       uvm_test_seq: sysrst_ctrl_ultra_low_pwr_vseq
+      run_opts: ["+test_timeout_ns=10_000_000_000"]
```

Comment:
Whoa! Do you really need such large timeout? If something goes wrong this could consume lots of resources doing nothing...

Reply:
The test in the regression is failing giving timeout error. I have added constraints to take the smaller value for the maximum number of times but its the 5% which picks up larger value.

Reply:
What is the most time consuming operation? Perhaps some other tuning is needed.